CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Chef Management Console Changelog
+
+The Chef Management Console has a public changelog that can be accessed from the hosted Chef Management Console at this link: [https://manage.chef.io/changelog](https://manage.chef.io/changelog)
```

Comment:
change log

(I know you could argue either way, but I've decided to go with "change log" :) )